tiles/src/main/java/com/protomaps/basemap/layers/Transit.java

```diff
@@ -20,6 +20,7 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
     // todo: exclude railway stations, levels
     if (sf.canBeLine() && (sf.hasTag("railway") ||
       sf.hasTag("aerialway", "cable_car") ||
+      sf.hasTag("man_made", "pier") ||
```

Comment:
This _should_ add missing pier lines, but needs more debugging.

Reply:
I finally did get this working see screenshot in https://github.com/protomaps/basemaps/pull/37#discussion_r1207412102.